src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
```

Comment:
The class 'Application' should have an access modifier. It's a good practice to explicitly define the access level for classes.